ToolsTest/Volodymyr/MSTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
```

Comment:
Remove this line as it is needless due to the same namespace is already defined and noone other uses `Assert` class.